rust/datafusion/README.md

```diff
@@ -19,11 +19,48 @@
 
 # DataFusion
 
-DataFusion is an in-memory query engine that uses Apache Arrow as the memory model. It supports executing SQL queries against CSV and Parquet files as well as querying directly against in-memory data.
+<img src="docs/images/DataFusion-Logo-Dark.svg" width="256"/>
+
+DataFusion is an extensible query execution framework, written in
+Rust, that uses [Apache Arrow](https://arrow.apache.org) as its
+in-memory format.
+
+DataFusion supports both an SQL and a DataFrame API for building
+logical query plans as well as a query optimizer and execution engine
+capable of parallel execution against partitioned data sources (CSV
+and Parquet) using threads.
+
+## Use Cases
```

Comment:
One other thing this page might benefit from is an example (e.g. look at how tokio does it https://crates.io/crates/tokio)

Perhaps we could just lift the nice example from https://docs.rs/datafusion/3.0.0/datafusion/ ?

Reply:
FYI I will do so as a follow on PR: https://github.com/apache/arrow/pull/9710